layouts/partials/header.html

```diff
@@ -14,12 +14,15 @@
 {{ if .RSSLink }}<link rel="alternate" type="application/rss+xml" title="{{ .Site.Title }} Feed" href="{{ .RSSLink }}">{{ end }}
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,400italic,700" type="text/css" media="all" />
 <link rel="stylesheet" href="{{ "css/style.css" | relURL }}" type="text/css" media="all" />
+<link rel="stylesheet" href="{{ "css/button.css" | relURL }}" type="text/css" media="all" />
 <script type="text/javascript" src="{{ "js/scripts.js" | relURL }}"></script>
+<script type="text/javascript" src="{{ "js/button.js" | relURL }}"></script>
 <link rel="shortcut icon" href="{{ "favicon.ico" | relURL }}">
 {{ template "_internal/google_analytics_async.html" . }}
 </head>
 <body class="body {{ if (.Site.Params.leftsidebar) or (.Params.leftsidebar) }}body-left-sidebar{{ else }}body-right-sidebar{{ end }} mobile" itemscope="itemscope" itemtype="http://schema.org/WebPage">
-	<div class="container container-outer">
+<button onclick="topFunction()" id="backTopBtn" title="Back to top">&#10148;</button>
```

Comment:
Perhaps you need to add the `btn` class and skip the parameters of `color`, `background-color`, `: hover` in css.
~the style suggestion~

Reply:
Actually, I wrote the style in `button.css`,you can see it [here](https://github.com/Vimux/Mainroad/pull/39/files/afa6f0ab302c98fea521727f12640905d9b50c50#diff-a64756b6f98a5dda16790718f7b4e467)

Reply:
@pennbay Of course. I wanted to say that it would be more consistent.